src/utils/time-code.js

```diff
@@ -24,8 +24,8 @@ export function timeCode<T>(label: string, codeAsACallback: () => T): T {
 
     // Some portion of users will have timing information sent. Limit this further to
     // only send a few labels per user.
-    const ga = self.ga;
-    if (ga && !(_timingsPerLabel[label] > MAX_TIMINGS_PER_LABEL)) {
+    const { ga } = self;
+    if (ga && _timingsPerLabel[label] <= MAX_TIMINGS_PER_LABEL) {
```

Comment:
When `_timingsPerLabel[label]` is `undefined` (the first time then), I think the result of this comparison is always `false`.

I think you need to set it to a temporary const before the condition, something like this:
```javascript
const sentTimings = _timingsPerLabel[label] || 0;
if (sentTimings < MAX_TIMINGS_PER_LABEL) {
  _timingsPerLabel[label] = sentTimings + 1;
  ...
}
``

Also note the operator `<` instead of `<=`. Otherwise I think it's sent (MAX_TIMINGS_PER_LABEL + 1) times.